src/main/java/at/hannibal2/skyhanni/features/misc/IslandLeaveJoinMsgs.kt

```diff
@@ -0,0 +1,144 @@
+package at.hannibal2.skyhanni.features.misc
+
+import at.hannibal2.skyhanni.SkyHanniMod
+import at.hannibal2.skyhanni.data.FriendAPI
+import at.hannibal2.skyhanni.data.GuildAPI
+import at.hannibal2.skyhanni.data.IslandType
+import at.hannibal2.skyhanni.data.PartyAPI
+import at.hannibal2.skyhanni.events.LorenzWorldChangeEvent
+import at.hannibal2.skyhanni.events.TabListUpdateEvent
+import at.hannibal2.skyhanni.utils.ChatUtils
+import at.hannibal2.skyhanni.utils.LorenzUtils
+import at.hannibal2.skyhanni.utils.StringUtils.matchMatcher
+import at.hannibal2.skyhanni.utils.StringUtils.removeColor
+import at.hannibal2.skyhanni.utils.repopatterns.RepoPattern
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent
+
+object IslandLeaveJoinMsgs {
+    private val config get() = SkyHanniMod.feature.misc.leaveJoinMsgs
+
+    private var players = mutableListOf<String>()
+    private var rawPlayersNew = mutableListOf<String>()
+    private var cleanPlayersNew = mutableListOf<String>()
```

Comment:
These are only used inside of the `onTabListUpdate` function why are they here and not local to the function?

Reply:
uhhhhhh
idk lmao